hw02/Progdata_Homework2.Rmd

```diff
@@ -0,0 +1,127 @@
+---
+title: "Hw2"
+author: "Neslihan Caliskan"
+date: "2/8/2021"
+output: 
+  html_document: default
+  github_document: default
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+
+```
+
+
+Load in your packages
+```{r}
+library(gapminder)
+library(tidyverse)
+library(dplyr)
+
+```
+
+Exercise 1
+```{r Exercise 1.1 & 1.2}
+gapminder::gapminder 
+
+#Exercise 1.1 & 1.2
+#In order to filter by multiple rows you need to use the "%in%" function!
+#You can add more criteria by inserting "&" in the middle.
+#You can select multiple columns simply by insterting a comma in between them! 
+gapminder %>% 
+  filter(country %in% c("Afghanistan","Australia","Austria") & year > 1970 & year < 1980) %>% 
+  select(country, gdpPercap) 
+
+```
+
+```{r Exercise 1.3}
+gapminder %>% 
+  mutate(diffLifeExp = lifeExp - lag(lifeExp)) %>% 
+ summarize(diff(diffLifeExp, differences = 1))
```

Comment:
Since the question is asking for entries that have experienced a drop in life expectancy, shouldn't it be diffLifeExp < 0 or something similar? I am not sure why you assign differences =1 here.